frontend/src/types.ts

```diff
@@ -699,6 +701,9 @@ export interface FunnelResult {
     type: 'Funnel'
 }
 
+export interface FunnelResultWithBreakdown extends Omit<FunnelResult, 'result'> {
+    result: FunnelStep[][]
```

Comment:
The type of this API response is flattened by `aggregateBreakdownResult` into a better type.